Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.html

```diff
@@ -90,6 +90,11 @@
           </tr>
         </tbody>
       </table>
+      <mat-paginator [length]="length"
+      [pageSize]="pageSize"
+      [pageSizeOptions]="[5]"
```

Comment:
make pageSizeOptions a variable as well.

Reply:
I should make it an array or define it in ts?